python/ray/train/tests/test_huggingface_trainer.py

```diff
@@ -46,6 +42,7 @@ def train_function(train_dataset, eval_dataset=None, **config):
     training_args = TrainingArguments(
         f"{model_checkpoint}-wikitext2",
         evaluation_strategy=config.pop("evaluation_strategy", "epoch"),
+        logging_strategy=config.pop("logging_strategy", "epoch"),
```

Comment:
evaluation_strategy and logging_strategy are not actually being configured right?

Reply:
They are set for the e2e tests (epochs here, steps in the other file) - we are missing a validation test for when they are mismatched though, will add one